pkg/cmd/validate.go

```diff
@@ -159,16 +160,37 @@ func (c *commandFlags) Run(cmd *cobra.Command, args []string) error {
 		return err
 	}
 
-	files, err := utils.FindFiles(args...)
-	if err != nil {
-		return err
+	var files []*os.File
```

Comment:
I think it'd be a smaller change to just filter `-` out from the list of files and note in a variable that we also want to validate stdin.